src/exercism/ResistorColor.java

```diff
@@ -0,0 +1,26 @@
+class ResistorColor {
+    int code = 0;      
```

Comment:
Code should not be an instance variable. It is only used in colorCode, so please initialize it there

Reply:
https://rootstack.com/en/blog/java-variables#:~:text=Local%20variables%20must%20be%20initialized,copy%20of%20the%20instance%20variables.